lib/matplotlib/backends/backend_svg.py

```diff
@@ -1119,10 +1119,9 @@ def _draw_text_as_text(self, gc, x, y, s, prop, angle, ismath, mtext=None):
             weight = fm.weight_dict[prop.get_weight()]
             if weight != 400:
                 font_parts.append(f'{weight}')
-            font_parts.extend([
-                f'{short_float_fmt(prop.get_size())}px',
-                f'{prop.get_family()[0]!r}',  # ensure quoting
-            ])
+            font_parts.append(f'{short_float_fmt(prop.get_size())}px')
+            font_parts.append(", ".join([f'{prop.get_name()!r}'] +
+                                        prop.get_family()))
```

Comment:
I'm not sure we need the `join` for two hard-coded elements, instead of placing them in a single f-string?